django/core/management/utils.py

```diff
@@ -153,3 +155,13 @@ def ignore(pattern):
         )
 
     return any(ignore(pattern) for pattern in normalize_path_patterns(ignore_patterns))
+
+
+def run_formatters(written_files):
+    """
+    Run the black formatter on the specified files.
+    """
+    if shutil.which("black") is None:
+        # Black is not installed.
+        return
+    subprocess.run(["black", "--", *written_files], capture_output=True)
```

Comment:
```suggestion
    black_path = shutil.which("black")
    if black_path:
        subprocess.run([black_path, "--", *written_files], capture_output=True)
```

Reply:
An actual opportunity to use a walrus! 😀

```python
    if black_path := shutil.which("black"):
         subprocess.run(
             [black_path, "--", *written_files],
             capture_output=True,
         )
```